AppDelegate.swift

```diff
@@ -0,0 +1,35 @@
+//
+//  AppDelegate.swift
+//  To-Do App
+//
+//  Created by Ananth Desai on 07/02/22.
+//
+
+import UIKit
+
+@UIApplicationMain
+class AppDelegate: UIResponder, UIApplicationDelegate {
+    var window: UIWindow?
+
+    func application(_: UIApplication, didFinishLaunchingWithOptions _: [UIApplication.LaunchOptionsKey: Any]?) -> Bool {
+        // Override point for customization after application launch.
+        UINavigationBar.appearance().barTintColor = UIColor(red: 0, green: 0 / 255, blue: 205 / 255, alpha: 1)
+
+        window = UIWindow(frame: UIScreen.main.bounds)
```

Comment:
You are not setting any root view controller. This will not run on devices below iOS 13.